frontend/src/pages/login_page/login_page.jsx

```diff
@@ -0,0 +1,7 @@
+import Form from '../../components/login_page/login';
+
+export default function Square() {
```

Comment:
let's use a better name than Square! it's hard to tell what this is meant to be used for from the name alone